src/lib/event_mapper/impl/general_aws_event_mapper.py

```diff
@@ -125,3 +135,24 @@ def to_message(self) -> dict:
         }
 
         return message
+
+
+class ExecutionInfoUrlMixin:
+    @staticmethod
+    def get_url(
+        resource_type: str,
+        region_name: str,
+        resource_name: str,
+        account_id: str = None,
+        run_id: str = None,
+    ) -> str:
+        """Returns the link to the particular resource run."""
+        url_mapping = {
+            types.GLUE_JOBS: f"https://{region_name}.console.aws.amazon.com/gluestudio/home?region={region_name}#/job/{resource_name}/run/{run_id}",
+            types.STEP_FUNCTIONS: f"https://{region_name}.console.aws.amazon.com/states/home?region={region_name}#/v2/executions/details/arn:aws:states:{region_name}:{account_id}:execution:{resource_name}:{run_id}",
+            types.LAMBDA_FUNCTIONS: f"https://{region_name}.console.aws.amazon.com/cloudwatch/home?region={region_name}#logsV2:log-groups/log-group/$252Faws$252Flambda$252F{resource_name}/log-events/",
+            types.GLUE_CRAWLERS: f"https://{region_name}.console.aws.amazon.com/glue/home?region={region_name}#/v2/data-catalog/crawlers/view/{resource_name}",
+            types.GLUE_DATA_CATALOGS: f"https://{region_name}.console.aws.amazon.com/glue/home?region={region_name}#/v2/data-catalog/databases/view/{resource_name}",
```

Comment:
For this PR, it's ok, but let's review this later - seems like it might depend on type of data catalog entity (database / table etc.)

Reply:
yeah, agree. it was also not crystal clear to me which link to choose for Glue Data Catalogs... I'll add a task to review it later